webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const SRC_DIR = path.join(__dirname, '/client/src');
+const DIST_DIR = path.join(__dirname, '/client/dist');
+
+module.exports = {
+    //mode: "development",
+    //devtool: 'none', 
+    entry: `${SRC_DIR}/index.jsx`,
+    output: {
+        filename: 'bundle.js',
+        path: DIST_DIR
+    },
+    module: {
+        rules: [
+            {
+                test: /\.jsx$/,
+                loader: 'babel-loader',
+                exclude: ["/node_modules"],
+                options: {
+                        presets: ['@babel/preset-react']
+                }
+            },
+            {
+                test: /\.css$/,
+                use: ['style-loader', 'css-loader']
+            }
+        ]
+    }
+
+}
```

Comment:
good job on setup. The last thing I would add is @babel/present-env to presets

Reply:
oh yeah. i send you a couple dms just now before reading this.

i guess what im learning is that even though the browser might be able to translate it, other browsers might now be able to.

so yeah, looks like it's a good idea to use always.

oh and fun fact, it replaces @babel/preset-es2015. (i think, at least that's what i heard a youtube say)

Reply:
*might not be able to